src/tests/ftest/launch.py

```diff
@@ -790,13 +789,37 @@ def run_tests(test_files, tag_filter, args):
         command_list.extend(tag_filter)
 
     # Run each test
+    skip_reason = None
     for test_file in test_files:
-        if isinstance(test_file["yaml"], str):
+        if skip_reason is not None:
+            # An error was detected running clean_logs for a previous test.  As
+            # this is typically an indication of a communication issue with one
+            # of the hosts, do not attempt to run subsequent tests.
+            if not report_skipped_test(
+                    test_file["py"], avocado_logs_dir, skip_reason):
+                return_code |= 64
+
+        elif not isinstance(test_file["yaml"], str):
+            # The test was not run due to an error replacing host placeholders
+            # in the yaml file.  Treat this like a failed avocado command.
+            reason = "error replacing yaml file placeholders"
+            if not report_skipped_test(
+                    test_file["py"], avocado_logs_dir, reason):
+                return_code |= 64
+            return_code |= 4
+
+        else:
             # Optionally clean the log files before running this test on the
             # servers and clients specified for this test
             if args.clean:
                 if not clean_logs(test_file["yaml"], args):
-                    return 128
+                    # Report errors for this skipped test
+                    skip_reason = "host communication error cleaning logs"
```

Comment:
I wonder if we should make it more clear here that this was cleaning logs in preparation for the test and not that it was the cleanup of this test's logs that was the problem.  I fear people will be looking for the logs assuming they got artifacted prior to the cleanup and the failure of it.
```suggestion
                    skip_reason = "host communication error cleaning out old logs prior to test run"
```

Reply:
That is clear.  I like it.